data/human/news.txt

```diff
@@ -72,6 +72,11 @@ news "merchant anywhere"
 			"Having both feet firmly on the ground doesn't make you take one step to completing your missions."
 			"The darkest hour is just before the departure."
 			"Don't look a gift shuttle in the engine outlets."
+			"Getting there is half the fun. Getting paid is the other half."
+			"Being a pilot is not all fun and games. We have angry customers, hostile pirates and asteroids to deal with."
+			"We are always a risk of being killed by faulty equipment, solar storms, asteroids, pirates or by angry customers. Ever wonder why you didn't just take a desk job?"
```

Comment:
```suggestion
			"We are always a risk of being killed by faulty equipment, solar storms, asteroids, pirates or angry customers. Ever wonder why you didn't just take a desk job?"
```

The second "by" is unnecessary, although you could probably keep it if you want to.